docker-compose.yaml

```diff
@@ -0,0 +1,29 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
```

Comment:
Mounting the entire current directory to `/app` can lead to potential issues with unwanted files being included in the container. Consider specifying only the necessary directories or files.